paraview/xmls/MergeTreeAutoencoder.xml

```diff
@@ -0,0 +1,653 @@
+      label="Normalized Wasserstein"
+      number_of_elements="1"
+      default_values="1">
+        <Documentation>
+          Choose between the normalizd Wasserstein or the usual Wasserstein metric. If merge trees are provided in input, they will automatically be converted to persistence diagrams if this option is disabled.
+        </Documentation>
+        <BooleanDomain name="bool"/>
+      </IntVectorProperty>
+    
+      <IntVectorProperty
+      name="NumberOfEncoderLayers"
+      command="SetNumberOfEncoderLayers"
+      label="Number of Encoder Layers"
+      number_of_elements="1"
+      default_values="1">
+        <Documentation>
+          Number of layers in the encoder (without counting the latent space layer).
+        </Documentation>
+      </IntVectorProperty>
+      
+      <IntVectorProperty
+      name="ScaleLayerAfterLatent"
+      command="SetScaleLayerAfterLatent"
+      label="ScaleLayerAfterLatent"
+      number_of_elements="1"
+      default_values="0"
+      panel_visibility="advanced">
+        <Documentation>
+          By default, the latent layer and the layer after it will have the same number of axes. This option allows to set the number of axes in the layer after the latent one as the average with the layer after it.
+        </Documentation>
+        <BooleanDomain name="bool"/>
+      </IntVectorProperty>
+      
+      <IntVectorProperty
+      name="InputNumberOfAxes"
+      command="SetInputNumberOfAxes"
+      label="Input Number of Axes"
+      number_of_elements="1"
+      default_values="16"
+      panel_visibility="advanced">
+        <Documentation>
+          Number of axes in the input layer.
+        </Documentation>
+      </IntVectorProperty>
+      
+      <DoubleVectorProperty
+      name="InputOriginPrimeSizePercent"
+      command="SetInputOriginPrimeSizePercent"
+      label="Input Origin Prime Size Percent"
+      number_of_elements="1"
+      default_values="15"
+      panel_visibility="advanced">
+        <Documentation>
+          Adjust the maximum size of the origin of the first output basis, given the total number of nodes in the input.
+        </Documentation>
+      </DoubleVectorProperty>
+      
+      <IntVectorProperty
+      name="LatentSpaceNumberOfAxes"
+      command="SetNumberOfAxes"
+      label="Latent Space Number of Axes"
+      number_of_elements="1"
+      default_values="2">
+        <Documentation>
+          Number of axes in the latent space.
+        </Documentation>
+      </IntVectorProperty>
+      
+      <DoubleVectorProperty
+      name="LatentSpaceOriginPrimeSizePercent"
+      command="SetLatentSpaceOriginPrimeSizePercent"
+      label="Latent Space Origin Prime Size Percent"
+      number_of_elements="1"
+      default_values="10"
+      panel_visibility="advanced">
+        <Documentation>
+          Adjust the maximum size of the origin of the latent output basis, given the total number of nodes in the input.
```

Comment:
-> "as a percentage of the total number of nodes in the input" (?)

Reply:
Done in the next commit!